src/meta/src/manager/notification.rs

```diff
@@ -41,6 +42,7 @@ struct Task {
     callback_tx: Option<oneshot::Sender<NotificationVersion>>,
     operation: Operation,
     info: Info,
+    hummock_version_delta: HummockVersionDelta,
```

Comment:
Could we just store `hummock_version_delta` in `info` int the first place, rather than doing the same L266?
I think the specific field `hummock_version_delta is `Task` is unnecessary.

Reply:
> Could we just store `hummock_version_delta` in `info` int the first place, rather than doing the same L266?
> I think the specific field `hummock_version_delta` in `Task` is unnecessary.

sorry. fixed.